tableau_exporter/tableau_status_exporter.py

```diff
@@ -88,9 +102,11 @@ def describe(self):
 
 def start_webserver(conf):
 
-    REGISTRY.register(TableauMetricsCollector(TokenManager(
-        conf['tableau_user'], conf['tableau_password'], conf['site'],
-        conf['server_host'], conf['api_version'])))
+    token_manager = TokenManager(
+        conf['server_host'], conf['api_version'], conf['site'],
+        conf.get('tableau_user'), conf.get('tableau_password'), conf.get('tableau_token_name'),
+        conf.get('tableau_token_secret'))
+    REGISTRY.register(TableauMetricsCollector(token_manager, verify_ssl=conf.get('verify_ssl', True)))
```

Comment:
I think defaulting this to true would be best practice, but understand if we want to avoid that for backwards compatibility.

Reply:
I'd vote for keeping backwards compatibility for now.